app/Commands/Dev/Geb/Repl.hs

```diff
@@ -114,25 +114,25 @@ checkTypedMorphism gebMorphism = Repline.dontCrash $ do
     Right _ -> printError (error "Checking only works on typed Geb morphisms.")
 
 runReplCommand :: String -> Repl ()
-runReplCommand input =
+runReplCommand input_ =
```

Comment:
Analogously with the `input` that somehow appeared in `Prelude` or somewhere in this PR and requires adjusting variable names all over the place. This is an unrelated (?) refactor.

Reply:
This change was moved to https://github.com/anoma/juvix/pull/2582